src/modules/endpoint/ContentLoadCriterion.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+import moment from 'moment-timezone'
+import Moment from 'moment'
+
+const MAX_CONTENT_AGE = 24
+
+export type ContentLoadCriterionType = {|
+  forceUpdate: boolean,
+  shouldRefreshResources: boolean
+|}
+
+export class ContentLoadCriterion {
+  _forceUpdate: boolean
+  _shouldRefreshResources: boolean
+  _peeking: boolean
+
+  constructor ({ forceUpdate, shouldRefreshResources }: ContentLoadCriterionType, peeking: boolean) {
```

Comment:
Why don't you add `peeking: boolean` to ContentLoadCriterionType? :thinking:

Reply:
Becaues ContentLoadCriterionType is used elsewhere without the peeking property.

Reply:
Shouldn't it just be `false` then?

Reply:
`peeking` can be true or false here.